src/server/rest/v1/service/security/TransactionSecurity.ts

```diff
@@ -110,4 +110,10 @@ export default class TransactionSecurity {
     }
     return filteredRequest;
   }
+
+  public static filterGetTransactions(request: any): HttpGetTransactionsRequest {
+    return {
+      Status: sanitize(request.Status),
```

Comment:
Should be part of the big http transaction type

Reply:
Done